.github/workflows/push-deploy-dry-run.yml

```diff
@@ -0,0 +1,27 @@
+name: (Dry Run) Deploy to WordPress.org
```

Comment:
Do we need a separate file for this? Couldn't we add a variable or something to the other one? Maybe like have it dry-run when creating a pre-release tag on GitHub?

Reply:
As far as I can tell, the only difference is line 27 so it essentially doubles our maintenance efforts.

Reply:
Agreed. I did initially look for a way to pass the variable conditionally but couldn't find one. I'll take another look and see if there's a way.
I like the idea of maybe using a dry run on plugin release build, as one option. Also, tbh, I'm not confident the dry-run helps us much. The only difference is the final SVN push is blocked. If the workflow fails for some reason before that, then it won't get pushed anyways. I'll also dig into this a bit more to see if that's indeed true.

Reply:
I've removed this workflow.